test/support/console_test_case.rb

```diff
@@ -195,12 +195,20 @@ def run_test_scenario cmd, test_info
         # result of `gets` return this exception in some platform
         rescue Timeout::Error
           assert_block(create_message("TIMEOUT ERROR (#{TIMEOUT_SEC} sec)", test_info)) { false }
+        rescue Test::Unit::AssertionFailedError
+          is_fail = true
```

Comment:
Can we align the name with what we have in protocol test case (`is_assertion_failure`)?